docs/user/filenodes.md

```diff
@@ -0,0 +1,56 @@
+# FlowForge File Nodes
+
+Projects running within FlowForge include a set of nodes that make it possible
+to store files safely regardless of the environment. For edge devices the files 
+are stored locally, containerized applications can read and write to persistent 
+storage via API calls to a file-server instance by using these nodes.
+
+*NOTE: This set of nodes allow you to read and write persistent files in a containerised
+environment only when coupled with a [file-storage](../install/file-storage/README.md) instance*
+
+
+## Nodes
+
+There are two nodes in the File Node collection:
+
+- `file` - A file node for writing to persistent storage
+- `file in` - A file node for reading from persistent storage
+
+## Install
+
+### FlowForge
+
+The File Nodes are installed automatically with FlowForge however in order for a 
+project to use them, the built-in Node-RED file nodes must first be disabled. 
+This is done by simply adding `10-file-js` in the **Exclude nodes by filename** 
+section of your FlowForge projects settings under  the **Palette** section
+
+### Standalone Node-RED
+
+These nodes can be installed through the Manage Palette option in the Node-RED
+editor, or on the command-line:
+
+```bash
+cd ~/.node-red
+npm install @flowforge/nr-file-nodes
+```
+
+This assumes the default location of the Node-RED user directory. If you are not
```

Comment:
If users just use the command palette this isn't a question right? If so; why recommend this?

Reply:
Actually, I have just checked & `npm install` is currently the only (recommended) way to install the file nodes - as it is not published in the flows library.

For now, I will remove the part about using the node-red library

@hardillb @knolleary should we publish `@flowforge/nr-file-nodes` on the flows lib?

Reply:
Done (next commit)

Reply:
Following discussion with Ben, these will not be published to flows & install information will be removed in next commit.